server/plugin/versions.go

```diff
@@ -7,11 +7,4 @@ var apiVersions = []string{
 }
 
 // CurrentAPIVersion is the newest API version
-var CurrentAPIVersion = apiVersions[0]
-
-var dataSchemaVersions = []string{
-	"v1",
-}
-
-// CurrentDataSchemaVersion is the newest data schema version
-var CurrentDataSchemaVersion = dataSchemaVersions[0]
+var CurrentApiVersion = apiVersions[0]
```

Comment:
I tent to remove this file completely. I hope we will never have more then two api version running and don't need a list for them.

Reply:
I agree with you.